docs/Workflow.md

```diff
@@ -489,160 +489,121 @@ is shown below.
 ## Data Model
```

Comment:
We should probably split this out into its own document. I understand now that the GpiiDataModel is just a legacy document, we should wrteit that a little to make this clear, since the first sentence still reads "This documentation describes the current data model used by the Personal Data Service"